api/src/main/resources/workbench.yaml

```diff
@@ -1236,6 +1236,9 @@ definitions:
       contactEmail:
         description: email address that can be used to contact the user
         type: string
+      contactEmailFailure:
+        description: Whether or not contact email could be added to verification list
+        type: boolean
```

Comment:
Just commenting here for my own learning. This file gets parsed into java code by the swagger code generators which is where the generated `Profile` class comes from.